### DISCUSSION TITLE
Do you wanna travel? Please write if you want!

### DISCUSSION THREAD

**Discussion by miranopari** [NAIST-SE/AttractingContributors#917] [2023 Attraction Contributions]
行きたい場所をかきこんでください！Please write your place where want to visit.
お願いします。
https://github.com/miranopari/travel/blob/main/README.md

**Comment by rfLENtlr** [comment]
I want to Waikiki beach!!

I sent a pull request.
Would you like to contribute to my repository as well?
https://github.com/rfLENtlr/AttractingContributors-OS
